docs/source/support/faq.rst

```diff
@@ -0,0 +1,183 @@
+            Working ~ Keep Ursula Online!
+
+    #. Ensure that your node uses the correct IP address can be accessed via port 9151 from an outside
+       connection eg. cell phone, other computer etc. by navigating to: ``https://<node_ip>:9151/status``
+
+    #. Ensure that your worker is bonded with your staker - ``nucypher stake list`` and check that
+       Worker is set correctly i.e. not ``0x0000``.
+
+    #. Ensure that your node is listed on the `Status Monitor Page <https://status.nucypher.network>`_ (this can take a few mins).
+
+       If your node is on the status monitor page but:
+
+        a. Does not have a green dot, you should ensure that your Ursula node can confirm activity.
+           Try locally running ``nucypher ursula confirm-activity``
+
+        b. Has a *Last Seen* value of ``No Connection to Node`` then there may be connectivity issues with your
+           node - redo the check in step #2.
+
+
+What's the best way to run Ursula in the background?
+----------------------------------------------------
+
+Either through `Docker <https://docs.nucypher.com/en/latest/guides/ursula_configuration_guide.html#running-an-ursula-with-docker>`_
+or `systemd <https://docs.nucypher.com/en/latest/guides/installation_guide.html#systemd-service-installation>`_.
+
+
+When installing on Docker, what do I input for <NETWORK NAME>?
+---------------------------------------------------------------
+
+For the *“Come and Stake It”* incentivized testnet, the network name is ``cassandra``.
+
+
+How can I check my current staking rewards?
+-------------------------------------------
+
+Run::
+
+    nucypher status stakers
+    >    --provider <your_geth_provider>
+    >    --network cassandra
+    >    --staking-address <your_staker_address>``
+
+It takes two periods to observe rewards - if you deposit in period 0 and start a worker in the same period
+(which calls ``confirmActivity()`` to receive work for the next period (1)), then in period 1 the Ursula
+node does work and will get rewarded in period 2 for the work performed in period 1.
+
+.. note::
+
+    `Restaking <https://docs.nucypher.com/en/latest/architecture/sub_stakes.html#re-staking>`_ is enabled by
+    default, and so NU inflation rewards are automatically restaked for you, and will be reflected in
+    the ``Staked`` value of the above command.
+
+
+How can I observe the settings (re-staking, winding down) for my stake?
+-----------------------------------------------------------------------
+
+Run::
+
+    nucypher status stakers
+    >    --provider <your_geth_provider>
+    >    --network cassandra
+    >    --staking-address <your_staker_address>``
+
+
+Can I extend the duration of my existing stake?
+--------------------------------------------------------------
+
+Yes, via the `prolong <https://docs.nucypher.com/en/latest/guides/staking_guide.html#prolong>`_ command.
+
+
+Why is the duration/enactment of my stake longer than the value I used during setup?
```

Comment:
imo we really should just change the way this is displayed. An increasing duration is too confusing.